docs/release-notes/8.15.asciidoc

```diff
@@ -1,6 +1,59 @@
 [[release-notes-header-8.15.0]]
 == 8.15
 
+[discrete]
+[[release-notes-8.15.1]]
+=== 8.15.1
+
+[discrete]
+[[known-issue-8.15.1]]
+==== Known issues
+
+// tag::known-issue-189676[]
+[discrete]
+.Tags appear in Elastic AI Assistant's responses
+[%collapsible]
+====
+*Details* +
+On August 1, 2024, it was discovered that Elastic AI Assistant's responses when using Bedrock Sonnet 3.5 may include `<antThinking>` tags, for example `<search_quality_reflection>` ({kibana-issue}189676[#189676]).
+
+
+====
+// end::known-issue-189676[]
+
+[discrete]
+[[features-8.15.1]]
+==== New features
+
+* Introduces a new feature for {elastic-defend} where Windows Image load events now include process protection status, making it easier to detect both legitimate and malicious PPL activity.
+* Allows Jamf data to be examined in the visual event analyzer ({kibana-pull}190965[#190965]).
+
+[discrete]
+[[enhancements-8.15.1]]
+==== Enhancements
+
+* Improves {elastic-defend} by reducing Malware Protection I/O and CPU usage when recently-written are subsequently executed. This change is for Windows endpoints only.
+* Makes several improvements to the detection and parsing of log samples that are uploaded to automatic import ({kibana-pull}190588[#190588], {kibana-pull}191502[#191502], {kibana-pull}190656[#190656], {kibana-pull}190046[#190046]).
+* Improves error handling for the Tines connector, and provides an option to use a webhook URL when connecting to the Tines API ({kibana-pull}191263[#191263]).
+
+[discrete]
+[[bug-fixes-8.15.1]]
+==== Bug fixes
+
+* Fixes an {elastic-defend} bug that affected CPU usage for Windows process events where the same executable is repeatedly launched, for example, during compilation workloads. With this fix, CPU usage is improved.
+* Fixes an {elastic-defend} bug that sometimes caused malware scan response actions to crash when they attempted to scan an inaccessible directory. 
+* Fixes an {elastic-defend} bug that sometimes caused Endpoint to report an incorrect version if it used an independent {agent} release.
+* Fixes an {elastic-defend} bug where the `process.thread.Ext.call_stack_final_user_module.protection_provenance_path` field might be populated with a non-path value. This fix is for Windows endpoints only.
+* Fixes an {elastic-defend} bug that can lead to Endpoint reporting `STATUS_ACCESS_DENIED` when attempting to open files for `GENERIC_READ`. Endpoint almost always recovered from this issue, but with this fix, it succeeds on the first try. This fix is for Windows endpoints only.
+* Fixes an {elastic-defend} regression that was introduced in 8.13.1, where security events did not populate the `user.name` field. This fix is for Windows endpoints only.
+* Fixes an {elastic-defend} bug where Endpoint sometimes missed file and network events on newer kernels that support ebpf. This only occurs if Endpoint fails to enable ebpf probes, falling back to kprobes. This fix is for Linux endpoints only.
```

Comment:
We might need to define this acronym. @gabriellandau can you verify that the definition is correct?

```suggestion
* Fixes an {elastic-defend} bug where Endpoint sometimes missed file and network events on newer kernels that support Extended Berkeley Packet Filter (eBPF). This only occurs if Endpoint fails to enable eBPF probes, and falls back to Kprobes. This fix is for Linux endpoints only.
```

Reply:
I'm okay either way.  I think users technical enough to dig through our logs to identify the issue would know what ebpf is.  That being said, defining it here is harmless can save curious users the double click -> right click -> "Search Google"

Reply:
I'll hold off on defining the acronym for now and will only commit the tense-related edits that I added above. Thanks, @gabriellandau